modules/vaos/app/controllers/vaos/limits_controller.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+require_dependency 'vaos/application_controller'
+
+module VAOS
+  class LimitsController < ApplicationController
+    def index
+      response = systems_service.get_facility_limits(
+        current_user,
+        limit_params[:facility_id],
+        limit_params[:type_of_care_id]
+      )
+
+      render json: VAOS::LimitSerializer.new(response)
+    end
+
+    private
+
+    def systems_service
+      VAOS::SystemsService.new
+    end
+
+    def limit_params
+      params.require(:facility_id)
+      params.require(:type_of_care_id)
+      params.permit(
+        :facility_id,
+        :type_of_care_id
+      )
+    end
```

Comment:
Mostly a nitpick, but wouldn't read a little bit better if you create methods for each of these, passing the method as arguments to your service?

```ruby
def facility_id
  params.require(:facility_id).permit(:facility_id)
end

def type_of_care_id
  params.require(:type_of_care_id).permit(:type_of_care_id)
end
```

Reply:
`require` returns the value(s) of the key(s) provided, so in this example, you'd be calling `permit` on a string.
And then `permit` still returns a `params` object.
So in this case you can just call `require` since we're not worried about mass assignment:
```ruby
def facility_id
  params.require(:facility_id)
end
```